pages/reference.md

```diff
@@ -0,0 +1,18 @@
+# Reference
+
+This page hosts a collection of reference documents to open datasets.
+
+## NEXRAD
+The Next Generation Weather Radar (NEXRAD) network, operated by the National Weather Service (NWS), Federal Aviation Administration (FAA) and United States Air Force, is openly available, with cloud-hosted versions of the datasets on both Amazon Web Services and Google Cloud.
+
+
+### NEXRAD Level 2 to Level 3 Processing
+
+- [PreprocessorAEL-V3Final10152009.pdf](../reference/nexrad/PreprocessorAEL-V3Final10152009.pdf)
+<iframe width="100%" height="800" src="../_downloads/5b5ba2eef5615ad129727fc593fc56d3/PreprocessorAEL-V3Final10152009.pdf"></iframe>
```

Comment:
And here we link from the final destination (the hash is file dependent and unique). One would have to create the docs first and pick the hash from the built resource to add the iframe-link.